frontend/src/components/ChatBox/ChatBoxInfoText.css

```diff
@@ -23,9 +24,10 @@
 	border-bottom: 1px solid var(--chat-reset-level-line-colour);
 	line-height: 0.1em;
 	margin: 0.6rem 0.1rem;
-	&:focus {
-		outline-offset: 0.5rem;
-	}
+}
+
+.chat-box-info.reset-level-text:focus-visible {
+	outline-offset: 0.5rem;
 }
```

Comment:
Not to do with linting, but this might have limited browser support. See [Peter's comment](https://github.com/ScottLogic/prompt-injection/pull/724#discussion_r1435133848).

Reply:
yes, it's not clear on this comment, should have included the removed lines too